crates/madara/src/commands/orchestrator.rs

```diff
@@ -1,39 +1,76 @@
-use madara_cli_common::logger;
-use madara_cli_config::madara::MadaraRunnerConfigMode;
+use madara_cli_common::{docker, logger, spinner::Spinner, PromptConfirm};
+use madara_cli_config::{
+    madara::{MadaraRunnerConfigMode, MadaraRunnerConfigSequencer, MadaraRunnerParams},
+    pathfinder::PathfinderRunnerConfigMode,
+    prover::ProverRunnerConfig,
+};
 use madara_cli_types::madara::MadaraMode;
 use xshell::Shell;
 
-use crate::commands;
+use crate::{
+    commands,
+    constants::{DEPS_REPO_PATH, MSG_BUILDING_IMAGE_SPINNER},
+};
 
-pub fn run(_shell: &Shell) -> anyhow::Result<()> {
+const ORCHESTRATOR_REPO_PATH: &str = "deps/orchestrator";
+const ORCHESTRATOR_DOCKER_IMAGE: &str = "orchestrator";
+const ORCHESTRATOR_COMPOSE_FILE: &str = "compose.yaml";
+
+pub fn run(shell: &Shell) -> anyhow::Result<()> {
     logger::new_empty_line();
     logger::intro();
 
-    let services: String = vec!["Madara", "SNOS", "Prover", "Pathfinder", "Anvil"]
-        .iter()
-        .map(|arg| format!("  ✅ {}", arg)) // You can replace "✅" with other emojis like "☑️" or custom checkboxes
-        .collect::<Vec<_>>()
-        .join("\n");
-    logger::note("AppChain configuration", services);
-
     // Collect Madara configuration
     let args_madara = MadaraRunnerConfigMode {
+        name: "Madara".to_string(),
         mode: Some(MadaraMode::Sequencer),
-        ..Default::default()
-    };
-
-    let shell = Shell::new().unwrap();
-    commands::madara::run(args_madara, &shell)?;
-
-    // Collect SNOS configuration
+        params: MadaraRunnerParams::Sequencer(MadaraRunnerConfigSequencer::default()),
+    }
```

Comment:
Wont this conflict with the madara db of another mode?
e.g: 
1. I first run Madara as Sequencer, run txs, and close it
2. Spin up AppChain (different mode, thus, different state)

I think this will use the same db, right?

Reply:
Actually not, cause they run using different compose files and the DBs are located in different paths but we need to handle that. Multiple DBs from different or the same mode. Probably this might happen for DBs for the same mode with different params.
The main issue here is that we need to dynamically modify some files like the compose file and it was not covered yet.